exec/java-exec/src/main/resources/rest/index.ftl

```diff
@@ -17,17 +17,56 @@
   <a href="/queries">back</a><br/>
   <div class="page-header">
   </div>
-  <div class="table-responsive">
-    <table class="table table-hover">
-      <tbody>
-        <#list model as stat>
-          <tr>
-            <td style="border:none;"><b>${stat.getName()}</b></td>
-            <td style="border:none; font-family: Courier;">${stat.getValue()}</td>
-          </tr>
-        </#list>
-      </tbody>
-    </table>
+
+  <#list model.getDrillbits() as drillbit>
+    <#if !drillbit.isVersionMatch()>
+      <div id="message" class="alert alert-danger alert-dismissable">
+        <button type="button" class="close" data-dismiss="alert" aria-hidden="true">&times;</button>
+        <strong>Drillbits in the cluster have different versions.</strong>
```

Comment:
Can we include more info? Something like:

Cluster version is x.y.z, but Drillbits in the cluster have other versions: a.b.c, d.e.f and i.j.k. Drill does not support clusters containing a mix of Drillbit versions.